lib/src/user.dart

```diff
@@ -118,15 +133,29 @@ enum UserState {
 
 /// The user identity associated with a [User]
 class UserIdentity {
-  /// The unique identifier for this [UserIdentity]
+  /// The unique identifier for this [UserIdentity].
   final String id;
 
-  /// The authentication provider defining this identity
+  /// The [AuthProviderType] defining this identity.
   final AuthProviderType provider;
 
   const UserIdentity._(this.id, this.provider);
 }
 
+class UserProfile {
+  final String? name;
+  final String? firstName;
+  final String? lastName;
+  final String? email;
+  final String? gender;
+  final String? birthDay;
+  final String? minAge;
```

Comment:
What about trying to parse min and max Age to int?

Reply:
.net has these as strings. So I left them the same